runbotics/runbotics-scheduler/src/scheduler-database/license/license.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { User as UserDecorator } from '#/utils/decorators/user.decorator';
+import { TenantInterceptor } from '#/utils/interceptors/tenant.interceptor';
+import { Logger } from '#/utils/logger';
+import { Controller, Get, Param, UseInterceptors } from '@nestjs/common';
+import { User } from '../user/user.entity';
+import { LicenseService } from './license.service';
+
+@UseInterceptors(TenantInterceptor)
+@Controller('/api/scheduler/tenants/:tenantId/licenses')
+export class LicenseController {
+    private readonly logger = new Logger(LicenseController.name);
+
+    constructor(private readonly licenseService: LicenseService) {}
+
+    @Get('plugins/available')
+    getAvailablePlugins(@UserDecorator() user: User) {
+        this.logger.log(
```

Comment:
We've decided to go with the default NestJS logs - https://all41.atlassian.net/wiki/spaces/RPA/pages/2987196431/RB+Guidelines#Logi-typu-info